SpatialGDK/Source/SpatialGDK/Private/Utils/InspectionColors.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/InspectionColors.h"
+
+#include "Containers/UnrealString.h"
+#include "Math/Color.h"
+#include "Math/UnrealMathUtility.h"
+#include "Misc/Char.h"
+
+#include "cstring"
+
+namespace SpatialGDK
+{
+	namespace
+	{
```

Comment:
code-style is non-indented namespaces?

Reply:
I realise this is an error from my original file (sorry 😄 )

Reply:
Done